tests/PoolTest.php

```diff
@@ -80,7 +79,7 @@ public function testPrefix()
             ->will($this->returnValue($loggerConfig));
 
         $factory = $this->getMock('\Phlib\Logger\Factory');
-        $logger  = $this->getMock('\Phlib\Logger\Gelf');
+        $logger  = $this->getMockBuilder('\Phlib\Logger\LoggerType\Stream')->disableOriginalConstructor()->getMock();
```

Comment:
Huh. I see why you changed this, there is no such class as `\Phlib\Logger\Gelf`..

Should probably change the `loggerConfig` above as well if we're trying to make this test less "wrong" (it still says `'type' => 'gelf'`).